### PR TITLE
Misc code improvements

### PR DESCRIPTION
**Pull request by dizzzz**
End-of-year Code cleanup, mechanical refactoring mostly.
Happy 2020 !

### CODE REVIEW

**Comment by dizzzz** [comment]
@adamretter ? @wolfgangmm ?

**Comment by adamretter** [comment]
@dizzzz I am slowly working my way through it... the change-set is huge ;-)

File: exist-core/src/main/java/org/exist/xupdate/Modification.java [hunk 1]
```diff
@@ -206,12 +205,12 @@ protected void declareVariables(XQueryContext context) throws XPathException {
...
-			context.declareNamespace(
-				entry.getKey(),
-				entry.getValue());
-		}
+        for (Map.Entry<String, String> stringStringEntry : namespaces.entrySet()) {
+            entry = stringStringEntry;
+            context.declareNamespace(
+                    entry.getKey(),
+                    entry.getValue());
+        }
```
Comment (adamretter) [inline exist-core/src/main/java/org/exist/xupdate/Modification.java]: I think this needs tidying up. `entry` is redundant, and then `stringStringEntry` should be renamed as `entry`.

File: exist-core/src/main/java/org/exist/storage/NativeBroker.java [hunk 2]
```diff
@@ -177,9 +173,9 @@
 
     private final Runtime run = Runtime.getRuntime();
 
-    private NodeProcessor nodeProcessor = new NodeProcessor();
+    private final NodeProcessor nodeProcessor = new NodeProcessor();
 
-    private IEmbeddedXMLStreamReader streamReader = null;
+    private final IEmbeddedXMLStreamReader streamReader = null;
```
Comment (adamretter) [inline exist-core/src/main/java/org/exist/storage/NativeBroker.java]: So `streamReader` will always be `null`?

seems that this should be removed, and `streamReaderNG` renamed to `streamReader`

File: exist-core/src/main/java/org/exist/xupdate/Modification.java (hunk 1 above)
Comment (dizzzz) [reply exist-core/src/main/java/org/exist/xupdate/Modification.java]: ah... good catch; i will fix

**Comment by dizzzz** [comment]
Thnx @adamretter  waiting for build to finish....

**Comment by dizzzz** [comment]
image

**Comment by dizzzz** [comment]
All green !

**Pull request merged**